05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
 - **Browser compatibility** solutions so there is consistent style across browsers
 - Creation of standard **CSS classes** which can be used to style advanced components of the user interface
 
-The most popular frameworks are Bootstrap and Foundation. Here's [an article](https://www.upwork.com/hiring/development/bootstrap-vs-foundation-which-framework-is-right-for-you/) comparing the two Frameworks. We'll be digging into Foundation, but the concepts apply to both frameworks.
+The most popular frameworks are Bootstrap and Foundation. Here's [an article](https://www.upwork.com/hiring/development/bootstrap-vs-foundation-which-framework-is-right-for-you/) comparing the two Frameworks. We'll be digging into Bootstrap, but the concepts apply to both frameworks.
 
-Foundation does provide a grid framework so developers can more quickly setup a layout for their site by simply, yet strategically, adding classes to HTML elements!
+Bootstrap does provide a grid framework so developers can more quickly setup a layout for their site by simply, yet strategically, adding classes to HTML elements!
 
 NOTE: _CSS Frameworks_, like Bootstrap and Foundation are different from _Front-end JavaScript frameworks_ like React, Angular and Ember.
 
-## Mobile First
-As browsing on phones has increasingly become popular, so has the philosophy of designing mobile first.  Mobile first is planning the design of your site for a very small screen before anything else.
-
-Doing so will help you concentrate on visual hierarchy and boiling your content down to what is most important.
-
-![mobile first visual](http://metamonks.com/wp-content/uploads/responsive-vs-mobile-first-webdesign-022-1024x689.png)
-
-That means creating separate wireframes for the different ways a user may interact with your site. Ex: A phone vertically, a phone horizontally, a large tablet in either direction, a laptop, a large-screen  television, I could go on forever!
-
-In an effort to not overwhelm ourselves, we’ll only be concerned on three generic screen sizes: small, medium and large. We refer to the user's visible area of a web page as the **viewport**.
-
-Think of small as a very narrow viewport, like your phone horizontally. Medium when you are split screening your laptop screen with your browser on one half and another application on the other. And large as your browser full-screen on your laptop.
-
 ## Grid Layout
-A grid system is an invisible foundation of web page layout, which is used to divide the web page multiple grids.
+A grid system is an invisible foundation of web page layout, which is used to divide the web page multiple grids.  While CSS Grid has reduced the need for a framework grid system there are advantages to a grid system like Bootstrap or Foundation.  One nice advantage to Bootstrap's grid system is that it [can be easier to make grids responsive to varying screen sizes.](https://getbootstrap.com/docs/4.1/layout/grid/#responsive-classes)
```

Comment:
This section on the Grid Layout now feels a little out of place -- It's before the "Include Bootstrap in your project" and there's no way to apply it.

Maybe this section can go into the part that lists the features of Bootstrap, around alerts and buttons.

Also, the grid images are pretty huge for something that we're not going to teach or introduce very deeply -- do you think we could just delete the images, since we have a link to say "feel free to explore it"?